### PR TITLE
OpenTelemetry ORM support information in FAQ.

### PR DESCRIPTION
**Pull request by balachandr**
OpenTelemetry support details.

### CODE REVIEW

**Comment by balachandr** [comment]
@odeke-em , please review and merge.

**Review by odeke-em — APPROVED** [review]
LGTM, thank you @balachandr! After this merge I’ll wait for a bit until my computer boots up then update the Google website too.

File: content/faq/_index.md
Comment (odeke-em) [inline content/faq/_index.md:81]: Minor nit: “ORMs support OpenTelemetry?”

File: content/faq/_index.md
Comment (odeke-em) [inline content/faq/_index.md:82]: Ditto here, please use “support” instead of “supports”

**Comment by balachandr** [comment]
Addressed your comment

**Pull request merged**
